ibc-core/ics03-connection/types/src/version.rs

```diff
@@ -291,7 +291,11 @@ mod tests {
         let tests: Vec<Test> = vec![
             Test {
                 name: "Compatible versions".to_string(),
-                versions: vec![Version::default().into()],
+                versions: vec![Version {
+                    identifier: "1".to_string(),
+                    features: vec!["ORDER_ORDERED".to_string(), "ORDER_UNORDERED".to_string()],
+                }
```

Comment:
feel free to use a `dummy_version` inside of test modules, like you used dummy features.